src/main/java/Duke.java

```diff
@@ -1,10 +1,90 @@
+
+import java.util.Scanner;
 public class Duke {
+    public static void printLines(){
+        System.out.println("--------------------------------------------------");
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?\n");
+
+        Scanner ReadInput = new Scanner(System.in);
+
+        Tasks[] task = new Tasks[100];
+        int taskCount = 0;
+
+        while(true){
+            String UserInput = ReadInput.nextLine();
+
+            if(UserInput.equalsIgnoreCase("bye")){
+                printLines();
+                System.out.println("Bye. Hope to see you again soon!\n");
+                printLines();
+                System.exit(0);
+            }
+
+            else if(UserInput.equalsIgnoreCase("list")){
+                printLines();
+                int indexNum = 1;
+                System.out.println("Here are the tasks in your list:");
+                for(int i = 0; i<taskCount; i++){
+                    System.out.println(indexNum + ". " + task[i]);
+                    indexNum++;
+                }
+                printLines();
```

Comment:
Hi, you can consider using the same level of abstraction instead of having multiple levels of abstraction.